gems/actionpack/CVE-2015-7576.yml

```diff
@@ -112,5 +112,5 @@ description: |
 patched_versions:
   - "~> 5.0.0.beta1.1"
   - "~> 4.2.5.1" 
-  - "~> 4.1.14.1"
+  - "~> 4.1.14, >= 4.1.14.1"
```

Comment:
I am assuming this is the correct syntax for a two-part constraint based on previous use [here](https://github.com/rubysec/ruby-advisory-db/blob/de575b873334e0a1eaab6d9a8dca3ea059bbc7a4/gems/actionpack/OSVDB-84243.yml#L22). A test run of `bundler-audit` seemed happy with the syntax.